Example/CustomTableViewCell.swift

```diff
@@ -0,0 +1,33 @@
+//
```

Comment:
Mind removing this header? The style of this project is to omit these copyright headers.